examples/tts/conf/zh/fastpitch_align_36finals_22050_pinyin.yaml

```diff
@@ -0,0 +1,253 @@
+# This config contains the default values for training FastPitch model with aligner using 22KHz sampling 
+# rate. If you want to train model on other dataset, you can change config values according to your dataset. 
+# Most dataset-specific arguments are in the head of the config file, see below.
+
+name: FastPitch
+
+train_dataset: ???
+validation_datasets: ???
+sup_data_path: ???
+sup_data_types: [ "align_prior_matrix", "pitch" ]
+
+# Default values from librosa.pyin
+pitch_fmin: 65.40639132514966
+pitch_fmax: 1986.977294921875
+
+# these frame-wise values depend on pitch_fmin and pitch_fmax, you can get values
+# by running `scripts/dataset_processing/tts/extract_sup_data.py`
+pitch_mean: ???  # e.g. 221.4948272705078 for SFbilingual dataset.
+pitch_std:  ???  # e.g.  64.6528930664063 for SFbilingual dataset.
+
+# Default values for dataset with sample_rate=22050
+sample_rate: 22050
+n_mel_channels: 80
+n_window_size: 1024
+n_window_stride: 256
+n_fft: 1024
+lowfreq: 0
+highfreq: null
+window: hann
+
+phoneme_dict_path: "scripts/tts_dataset_files/zh/36finals/pinyin_dict_nv23.05.txt"
```

Comment:
yeah, good question. I was also thinking of keeping only 36-final Pinyin and its IPA conversion. But would like to seek you guys opinion on that. @rlangman @redoctopus @yuekaizhang. Based on my experiments (see my demo pages linked in the t5t email), the IPA symbol set derived from the 36-final pinyin demonstrates superior sound quality overall, exhibiting greater naturalness and prosody compared to the other three setups. However, it is worth noting that the remaining three symbol sets are closely approaching the same level of performance.

Reply:
If you think there is good enough documentation that users would be able to reproduce the other configs, I'm in favor of just keeping the one.

Reply:
right, I can keep the IPA symbol set derived from 36-final pinyin as the final solution. But if doing so, our old model checkpoints based on the 24-final Pinyin symbol set have to be deprecated in a release or two. I can add a deprecation warning for those tokenizer/g2p relying on 24-final pinyin.

Reply:
After recent discussions, I also think we should stick to just the 1 config file with our best known config with documentation on how to edit it to reproduce the others.

If we want to keep around the 24-final until it can be deprecated, that sounds good to me.